test/frontend/pageloader_test.dart

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+@Tags(const ['aot'])
+@TestOn('browser')
+import 'package:angular2/angular2.dart';
+import 'package:angular_test/angular_test.dart';
+import 'package:pageloader/objects.dart';
+import 'package:test/test.dart';
+
+@AngularEntrypoint()
+void main() {
+  tearDown(() => disposeAnyRunningTest());
```

Comment:
`disposeAnyRunningTest` is not a reference that can change

`tearDown(disposeAnyRunningTest);`